examples/vumps/vumps_hubbard_extended.jl

```diff
@@ -1,11 +1,13 @@
 using ITensors
 using ITensorInfiniteMPS
 
-include(
-  joinpath(
-    pkgdir(ITensorInfiniteMPS), "examples", "vumps", "src", "vumps_subspace_expansion.jl"
-  ),
-)
+base_path = joinpath(pkgdir(ITensorInfiniteMPS), "examples", "vumps", "src")
+src_files = [
+  joinpath(base_path, "vumps_subspace_expansion.jl"), joinpath(base_path, "entropy.jl")
+]
+for f in src_files
+  include(f)
+end
```

Comment:
```suggestion
src_files = [
  "vumps_subspace_expansion.jl",
  "entropy.jl",
]
for f in src_files
  include(joinpath(base_path, f))
end
```

Reply:
I prefer the original just because it allows a user to add their own files to include in `src_files` without having to deal with `base_path`. If you still prefer this cleaner suggested version I'm happy to change it though

Reply:
I would rather use the simpler version, users can edit it as they see fit.